docs/Authorization/2-legged-oauth/UsingAServiceAccount.md

```diff
@@ -390,16 +390,18 @@ public class ServiceAccountSample {
 
             // Form and sign the JWT
             Map<String, Object> headerClaims = new HashMap<String, Object>();
-            headerClaims.put("kid", credentials.get("assertionHeaderData").get("kid"));
-            headerClaims.put("alg", credentials.get("assertionHeaderData").get("alg"));
+            JSONObject jsonObject = (JSONObject)credentials.get("assertionHeaderData");
```

Comment:
Let's give the variables meaningful names. 

```suggestion
            JSONObject headerData = (JSONObject)credentials.get("assertionHeaderData");
```

Reply:
I was trying them in my local to understand the 2legged oauth